src/main/kotlin/com/avail/interpreter/primitive/modules/P_VisibleAtoms.kt

```diff
@@ -43,11 +43,11 @@ import com.avail.descriptor.types.IntegerRangeTypeDescriptor.wholeNumbers
 import com.avail.descriptor.types.SetTypeDescriptor.setTypeForSizesContentType
 import com.avail.descriptor.types.TypeDescriptor.Types.ATOM
 import com.avail.exceptions.AvailErrorCode.E_LOADING_IS_OVER
-import com.avail.interpreter.execution.AvailLoader
-import com.avail.interpreter.execution.Interpreter
 import com.avail.interpreter.Primitive
 import com.avail.interpreter.Primitive.Flag.CanInline
 import com.avail.interpreter.Primitive.Flag.ReadsFromHiddenGlobalState
+import com.avail.interpreter.execution.AvailLoader
+import com.avail.interpreter.execution.Interpreter
 
 /**
  * **Primitive**: Answer every [true name][A_Atom] visible in the
```

Comment:
`&#32;`

Reply:
I found a more aggressive set of regular expressions for finding 1) spaces in links and 2) long lines. I iterated them until stasis. I visited every long line and reflowed. I found a couple hundred more sites and patched them. I can't find any other spaces in links or long comment lines at this point.